### PR TITLE
Completed react-hot-button-component

### PR DESCRIPTION
**Pull request by bbrown949**
https://github.com/bbrown949/c0523-code-solutions/assets/117103446/39b471bd-8b73-4879-89dd-4dc1eb456a1a

### CODE REVIEW

File: react-hot-button-component/my-app/src/HotButton.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    temperature = 'cold';
+  } else if (timesClicked < 9) {
+    temperature = 'luke-warm';
+  } else if (timesClicked < 12) {
+    temperature = 'warmer';
+  } else if (timesClicked < 15) {
+    temperature = 'hot';
+  } else if (timesClicked < 18) {
+    temperature = 'on-fire';
+  }
```
Comment (shawnkost) [inline react-hot-button-component/my-app/src/HotButton.js:22]: You never initialize your `temperature` variable

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change here Bryan!

**Review by shawnkost — APPROVED** [review]
👍🏻

File: react-hot-button-component/my-app/src/HotButton.js (hunk 1 above)
Comment (bbrown949) [reply react-hot-button-component/my-app/src/HotButton.js:24]: fixed!

**Pull request merged**